iidm/iidm-test/src/main/java/com/powsybl/iidm/network/test/TerminalMockExt.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.test;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Load;
+import com.powsybl.iidm.network.Terminal;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class TerminalMockExt extends AbstractExtension<Load> {
+
+    private Terminal terminal;
+
+    public TerminalMockExt(Load load) {
+        super(load);
+        terminal = load.getTerminal();
+    }
+
+    @Override
+    public String getName() {
+        return "terminalMock";
+    }
+
+    public Terminal getTerminal() {
+        return terminal;
+    }
+
+    public TerminalMockExt setTerminal(Terminal terminal) {
+        this.terminal = terminal;
```

Comment:
Is null allowed here? Even you are in iidm-test, you're not in the test folder and you should assert that terminal is not null, if null values are not allowed.

Reply:
yes i will add it